.github/workflows/linux.yml

```
@@ -68,6 +68,7 @@ jobs:
     container:
       image: dealii/dependencies:jammy
```

```suggestion
      image: dealii/dependencies:jammy
      options: --user root
```
You will need to run the image as root user to get rid of the permission denied errors (see [here](https://github.com/dealii/dealii/actions/runs/5125913163/jobs/9219713296?pr=15285))
```
mkdir: cannot create directory ‘build’: Permission denied
```